lms/static/scripts/postmessage_json_rpc/server/test/index-test.js

```diff
@@ -0,0 +1,80 @@
+import { startRpcServer, getSidebarWindow, $imports } from '../index';
+
+describe('postmessage_json_rpc/server', () => {
+  let FakeServer;
+  let FakeRegister;
+
+  beforeEach(() => {
+    FakeRegister = sinon.stub();
+
+    FakeServer = sinon.stub().returns({
+      register: FakeRegister,
+      sidebarWindow: 'FakeSidebarWindow',
+    });
+
+    $imports.$mock({
+      './server': FakeServer,
+    });
+  });
+
+  afterEach('remove the server config from the document', () => {
+    $imports.$restore();
+  });
+
+  describe('#startRpcServer', () => {
```

Comment:
```suggestion
  describe('startRpcServer', () => {
```

There are many existing tests that do this wrongly but `#` should be reserved for instance properties and methods (following JSDoc conventions). If you're just testing a function, just use the function name without a `#` prefix.

Reply:
This is my new "RCP", lol. I'm gonna make a ticket to clean this up because this has confused me a few times..